packages/substrate/src/__test__/utils/getBasicFee.test.ts

```diff
@@ -0,0 +1,91 @@
+import type { XcmMultiAssetIdType } from '@buildwithsygma/core';
+import { FeeHandlerType } from '@buildwithsygma/core';
+import type { ApiPromise } from '@polkadot/api';
+import { u128, Option } from '@polkadot/types';
+import { TypeRegistry } from '@polkadot/types/create';
+import { BN } from '@polkadot/util';
+
+import { getBasicFee } from '../../utils/getBasicFee.js';
+
+const registry = new TypeRegistry();
+
+describe('Substrate - getBasicFee', () => {
+  const validXcmMultiAssetId: XcmMultiAssetIdType = {
+    concrete: {
+      parents: 1,
+      interior: {
+        x3: [
+          {
+            parachain: 2004,
+          },
+          {
+            generalKey: [5, '0x12345'],
+          },
+          {
+            generalKey: [4, '0x1234'],
+          },
+        ],
+      },
+    },
+  };
+
+  it('should return the basic fee', async () => {
+    const mockFee = '100000';
+    const mockFeeBN = new BN(mockFee);
+    const rawResult = new Option(registry, u128, mockFee);
+
+    const api: ApiPromise = {
```

Comment:
would it make sense to add the mock of ApiPromise either to a mock file or to the `setupTest` file directly? In doing so we could avoid repeating the implementation and just importing the mock. Wdyt?

Reply:
@wainola I believe we should add that in a separate PR as an improvement

Reply:
I agree @saadjhk, let us just create an issue for this.

Reply:
created #390 , further description can be provided later